### ISSUE TITLE
[Entity Span QA] The shape of logits is not consistant with the shape of labels

### ISSUE THREAD

**Issue by h-peng17** [studio-ousia/luke#58]
Hi~ Thank you for your released code. But when I run the Entity Span QA task, I get the error:
![image](https://user-images.githubusercontent.com/39556019/111987319-0b65e500-8b4a-11eb-9da8-091f5d901f1b.png)
It was invoked by this line 
![image](https://user-images.githubusercontent.com/39556019/111987369-1f114b80-8b4a-11eb-97c0-619829f277f7.png)
In my understanding, logits is every token's propbility, but labels is only entities' labels, so maybe there is an error?

**Comment by ikuyamada** [comment]
Hi!
Thanks for using LUKE :) 
The logits variable contains the logits corresponding to entities.
At the following line, `entity_hidden_states` contains the output representations of entities. `encoder_outputs[0]` contains the output representations of tokens.
https://github.com/studio-ousia/luke/blob/master/examples/entity_span_qa/model.py#L39

**Comment by h-peng17** [comment]
I got it. Thank you! But if I want to use bert/roberta encoder, they don't have entity representations as luke, how can i change the code? Should I select the corresponding entity representation in some position?

**Comment by ikuyamada** [comment]
The code used for the ReCoRD task is specific to LUKE. If you need to use the same code for BERT/RoBERTa, maybe you can compute the entity representations based on the output word representations of BERT/RoBERTa.

**Comment by h-peng17** [comment]
Got it. Thank you~
